### PR TITLE
templates: Add LTP open posix template

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Add ltp-open-posix-template to run functional group tests

Signed-off-by: lakshmipathi.g <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
Depends on https://github.com/kernelci/test-definitions/pull/7/commits

**Comment by khilman** [comment]
For this to be tested on staging, the new templates need to be in `config/lava/ltp`

**Comment by khilman** [comment]
After moving files into `config/lava/ltp`, I tested this locally to generate jobs for lab-baylibre, and it the job fails for various reasons, one of which is missing `make` command: https://lava.baylibre.com/scheduler/job/701290#L1216

Am I missing an updated rootfs?

**Comment by Lakshmipathi** [comment]
Okay will move them to `config/lava/ltp`.  Yes, rootfs PR under review. We need all below changes.

buster-ltp rootfs PR: https://github.com/kernelci/kernelci-core/pull/599
ltp-open-posix PR : https://github.com/kernelci/test-definitions/pull/7/commits
test-configs.yml PR: https://github.com/kernelci/kernelci-core/pull/604

Sample LAVA job with amd64: https://lava.collabora.co.uk/scheduler/job/3166129
Just now Started arm64 build: https://bot.staging.kernelci.org/job/rootfs-builder/10/

File: config/lava/ltp/ltp-open-posix.jinja2 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- test:
+    name: {{ plan }}
+    timeout:
+      minutes: {{ job_timeout }}
+    definitions:
+       - repository: https://github.com/kernelci/test-definitions
+         from: git
+         revision: staging.kernelci.org
```
Comment (gctucker) [inline config/lava/ltp/ltp-open-posix.jinja2:8]: Please use the `kernelci.org` branch in this PR, the staging branch should only be used on staging.

**Comment by Lakshmipathi** [comment]
Sample Results from staging:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20210416.0/plan/ltp-timers/
https://staging.kernelci.org/test/plan/id/6078e0a20bfe3738d87747c2/
https://staging.kernelci.org/test/plan/id/6078df4f4ca8b9707a7747ef/

**Review by gctucker — APPROVED** [review]

**Pull request merged**